apps/storybook-react/tailwind.css

```diff
@@ -3,36 +3,43 @@
 @tailwind utilities;
 
 @font-face {
-  font-family: 'Euclid Circular B'; /* This should match the font-family in typography.ts */
+  font-family: 'CentraNo1';
   font-style: normal;
   font-weight: 400;
-  src: url('fonts/EuclidCircularB-Regular.woff2') format('woff2');
+  src: url('fonts/CentraNo1/CentraNo1-Book.woff2') format('woff2');
 }
 
 @font-face {
-  font-family: 'Euclid Circular B';
+  font-family: 'CentraNo1';
   font-style: italic;
   font-weight: 400;
-  src: url('fonts/EuclidCircularB-RegularItalic.woff2') format('woff2');
+  src: url('fonts/CentraNo1/CentraNo1-BookItalic.woff2') format('woff2');
 }
 
 @font-face {
-  font-family: 'Euclid Circular B';
+  font-family: 'CentraNo1';
   font-style: normal;
   font-weight: 500;
-  src: url('fonts/EuclidCircularB-Medium.woff2') format('woff2');
+  src: url('fonts/CentraNo1/CentraNo1-Medium.woff2') format('woff2');
 }
 
 @font-face {
-  font-family: 'Euclid Circular B';
+  font-family: 'CentraNo1';
   font-style: italic;
   font-weight: 500;
-  src: url('fonts/EuclidCircularB-MediumItalic.woff2') format('woff2');
+  src: url('fonts/CentraNo1/CentraNo1-MediumItalic.woff2') format('woff2');
 }
 
 @font-face {
-  font-family: 'Euclid Circular B';
+  font-family: 'CentraNo1';
   font-style: normal;
   font-weight: 700;
-  src: url('fonts/EuclidCircularB-Bold.woff2') format('woff2');
+  src: url('fonts/CentraNo1/CentraNo1-Bold.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'CentraNo1';
+  font-style: italic;
+  font-weight: 700;
+  src: url('fonts/CentraNo1/CentraNo1-BoldItalic.woff2') format('woff2');
 }
```

Comment:
Adding `CentraNo1-BoldItalic` to match mobile and portfolio

- Replaced *EuclidCircularB* with *CentraNo1*
- Added a new nested folder to support future additions like *MMSans* and *MMPoly*
- Replaced `Regular` with `Book`
- Only existing font files are listed

![Screenshot 2025-03-21 at 2 52 54 PM](https://github.com/user-attachments/assets/5a5c45f7-dbcf-4930-bd66-336e355f3c3b)


- Aligns with what we have in extension

![Screenshot 2025-03-21 at 2 55 03 PM](https://github.com/user-attachments/assets/b675aae3-c564-4798-ac55-85075cb0c246)